app/services/cards.ts

```diff
@@ -0,0 +1,259 @@
+import axios from 'axios';
+import config from '../../config';
+import { externalApiError } from '../errors';
+
+export const BASE_URL = config.hearthstone.baseURL as string;
+export const KEY = config.hearthstone.secret as string;
+
+const card = axios.create({
+  baseURL: BASE_URL,
+  responseType: 'json',
+  headers: {
+    'x-rapidapi-host': 'omgvamp-hearthstone-v1.p.rapidapi.com',
+    'x-rapidapi-key': KEY
+  }
+});
+
+export interface Card {
```

Comment:
maybe you can create a folder called interfaces and create all interfaces inside

Reply:
created fold called "interfaces" for all interfaces